classes/SpecificPrice.php

```diff
@@ -211,7 +214,7 @@ public static function getIdsByProductId($id_product, $id_product_attribute = fa
     /**
      * score generation for quantity discount
      */
-    protected static function _getScoreQuery($id_product, $id_shop, $id_currency, $id_country, $id_group, $id_customer)
+    protected static function _getScoreQuery($id_product, $id_shop, $id_currency, $id_country, $id_group, $id_customer, $id_zone)
```

Comment:
`$id_zone` parameter is not used in this method

Reply:
it is called dynamically